src/model.jl

```diff
@@ -134,61 +133,61 @@ function EMB.variables_capex(m, 𝒩, 𝒯, 𝒫, modeltype::AbstractInvestmentM
 end
 
 """
-    EMB.constraints_capacity_installed(m, n::EMB.Node, 𝒯, modeltype::AbstractInvestmentModel
+    EMB.constraints_capacity_installed(
+        m,
+        n::EMB.Node,
+        𝒯::TimeStructure,
+        modeltype::AbstractInvestmentModel,
+    )
 
-Set capacity-related constraints for nodes `𝒩` for investment time structure `𝒯`:
-* bounds
-* binary for BinaryInvestment
-* link capacity variables
+When the modeltype is an investment model, the function introduces the related constraints
+for the capacity expansion. The investment mode and lifetime mode are used for adding
```

Comment:
```suggestion
The investment mode and lifetime mode are used for adding
```

Reply:
NP, I think I missed the context and worded it like it was the first sentence, but since it's not we can keep as is.